scss/mixins/typo.scss

```
@@ -0,0 +1,104 @@
+@mixin headingDisplay($color: $darkBg) {
+    color: $color;
+    font-family: "Sen";
```

Định nghĩa thêm các font dự phòng nữa em nhé
```suggestion
    font-family: "Sen", Arial, san-serif....;
```

---

Oce a